### PR TITLE
Use more AssertJ features

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: okhttp-tests/src/test/java/okhttp3/EventListenerTest.java [hunk 1]
```diff
@@ -267,34 +256,30 @@ private void assertBytesReadWritten(RecordingEventListener listener,
 
     if (requestHeaderLength != null) {
       RequestHeadersEnd responseHeadersEnd = listener.removeUpToEvent(RequestHeadersEnd.class);
-      assertThat("request header length", responseHeadersEnd.headerLength, requestHeaderLength);
+      Assert.assertThat("request header length", responseHeadersEnd.headerLength, requestHeaderLength);
     } else {
```
[the remark this answers is not in the document]
Comment (swankjesse) [reply okhttp-tests/src/test/java/okhttp3/EventListenerTest.java]: It's using some Hamcrest advanced features and the AssertJ replacement isn’t mechanical!

**Comment by swankjesse** [comment]
@yschimke since I cut a new release our builds are all breaking ’cause `dependency:resolve` can’t find a snapshot for 3.15.0. Any idea why Maven isn’t looking in the current project for this dependency? 

I think it’s due to this thing in our `.travis.yml` file:

```
install:
  - ./mvnw dependency:resolve -B
```

**Comment by swankjesse** [comment]
workaround! https://github.com/square/okhttp/pull/4721

**Comment by yschimke** [comment]
The resolve is to front load these so the 3 later builds are just doing the incremental work instead of downloading jars, noise slowness etc.  Your fix looks great.  Sorry about the break.

**Pull request merged**